LICENSE.rst

```diff
@@ -0,0 +1,30 @@
+BSD 3-Clause License
```

Comment:
We will use the copyright from this file, `Copyright (c) 2024, The Trustees of Columbia University
in the City of New York.` but not the rest, so I suggest to copy this copyright statement nad put it in the other license file (don't delete anything in that file), then delete this whole file.